.pre-commit-config.yaml

```diff
@@ -5,7 +5,9 @@ repos:
     rev: v4.6.0
     hooks:
       - id: trailing-whitespace
-        exclude: ^pineappl_cli/tests/
+        # `.rs` files are taken care of by `cargo fmt`
+        # `.github/workflows/release.yml` contains vital spaces
```

Comment:
shouldn't we rather move that part out of the `.yml` file?

Actually, having looked at the file, it is much too complicated I'd say - can't we split it into smaller snippets and scripts which are executed in the correct order inside the workflow?